### ISSUE TITLE
Password set via UI gets overwritten by initialize_django playbook

### ISSUE THREAD

**Issue by gamuniz** [ansible/tower-operator#869]
##### ISSUE TYPE
 - Bug Report

##### SUMMARY

Setting the admin password via UI to the database gets overwritten on reconciliation.

##### ENVIRONMENT
* AWX version: latest
* Operator version: devel
* Kubernetes version: 1.22
* AWX install method: AKS

##### STEPS TO REPRODUCE
1. deploy awx-operator
2. change the admin password via UI
3. wait until loop
4. attempt to use changed password to log in

##### EXPECTED RESULTS
Password which was changed is used.

##### ACTUAL RESULTS
original admin_password is set again. Can't log in.

**Comment by fosterseth** [comment]
yeah I can replicate the issue -- for step 3 (wait for loop) you can delete the awx pods to trigger the operator to recreate them. Upon recreation, the admin password is changed back whatever the k8s secret is

@rooftopcellist how necessary is this update password task?

https://github.com/ansible/awx-operator/blob/1b41d945e6d2fc616f8b9234c0240cab89ac89da/roles/installer/tasks/initialize_django.yml#L16

ideally we'd only create the password once, when we create the admin user.

**Comment by rooftopcellist** [comment]
@fosterseth This task was added for the following scenario:

* User changes the password in the admin_password secret after deploying AWX
* User expects that the password is updated for the user specified (admin by default) in the database

If the admin user already exists, as in the above case, it will not be updated (edit: it will not be updated without that task).  Another thing to consider is that the user could accidentally delete their `awx-admin-password` secret, then have no (easy) way to log in.  

I would be open to changing this behavior.

**Comment by gamuniz** [comment]
@rooftopcellist we are seeing different behavior than what you're saying. regardless of whether or not the user exists the password gets reset. should there be a conditional or toggle to flip this behavior?

**Comment by rooftopcellist** [comment]
The current behavior is undesirable because a user may want to change the admin password for security reasons, but currently it would be reset to the value in the awx-admin-password secret upon the next reconcilation loop.

I agree that this should be removed @gamuniz , here is a PR to do so: https://github.com/ansible/awx-operator/pull/874

**Comment by hesmithrh** [comment]
moving to our blocked state until we have a clear action on Controller side
